rust_icu_uloc/src/lib.rs

```diff
@@ -121,19 +244,23 @@ impl ULoc {
         let mut status = common::Error::OK_CODE;
 
         let mut accept_list_cstrings: Vec<ffi::CString> = vec![];
-        // This is mutable only to satisfy the missing `const`s in the ICU4C API.
-        let mut accept_list: Vec<*const raw::c_char> = accept_list
+        let accept_list: Result<Vec<*const raw::c_char>, _> = accept_list
```

Comment:
You probably want to add a comment to explain why this exists.

Reply:
Reverted.